src/components/UserSelector.tsx

```diff
@@ -72,9 +70,7 @@
             <a
               key={user.id}
               href={`#user-${user.id}`}
-              onClick={() => {
-                onChange(user);
-              }}
+              onClick={() => handleDispatchSelectedUser(user)}
```

Comment:
Consider adding keyboard accessibility features, such as handling 'Enter' key presses, to improve the user experience for keyboard users.